keps/sig-node/3695-pod-resources-for-dra/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: Extend the PodResources API to include resources allocated by DRA
+kep-number: 3695
+authors:
+  - "@moshe010"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: implementable
+reviewers:
+  - "@fromanirh"
+  - "@swatisehgal"
+approvers:
+  - "@sig-node-leads"
+see-also:
+  - "keps/sig-node/2403-pod-resources-allocatable-resources"
+  - "keps/sig-node/606-compute-device-assignment/"
+  - "keps/sig-node/2043-pod-resource-concrete-assigments/"
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.27"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.27"
+  beta: "v1.28"
+  stable: "v1.29"
```

Comment:
minor: these milestone doesn't seem to match the expected graduation time spelled out in the KEP. I guess PRReviewers (or approvers or both) will comment with suggestions.

Reply:
What do you mean, where else are these spelled out?

Reply:
Sorry, "spelled out" was bad english in my case. I was thinking to the beta -> GA graduation. In the README, among the graduation criterias, it was listed ~1 year for feedback. Yet here is planned to go GA in 1.29, which doesn't seem to match the 1y time interval. We can totally fix this later though.

Reply:
Yeah, I see. As @cici37 points out, this feature is dependent on DRA itself, so it' hard to gauge when this can go beta / GA since DRA will have to graduate through these stages first.